base/docs/helpdb.jl

```diff
@@ -11455,9 +11455,9 @@ If `pkg` has a URL registered in `Pkg.dir("METADATA")`, clone it from that URL o
 Pkg.clone(pkg)
 
 doc"""
-    checkout(pkg, [branch="master"])
+    checkout(pkg, [branch="master"]; merge=true, pull=true)
 
-Checkout the `Pkg.dir(pkg)` repo to the branch `branch`. Defaults to checking out the "master" branch. To go back to using the newest compatible released version, use `Pkg.free(pkg)`
+Checkout the `Pkg.dir(pkg)` repo to the branch `branch`. Defaults to checking out the "master" branch. To go back to using the newest compatible released version, use `Pkg.free(pkg)`\ . Changes are merged (fast-forward only) if the keyword argument ``merge == true``, and the latest version is pulled from the upsream repo if ``pull == true``\ .
```

Comment:
Note that in the julia source the docstring is formatted as markdown by default, it's converted to RST by the script `doc/genstdlib.jl` which you'd need to run with a just-built julia.

So the backslash-space before period is not necessary in this copy, and the code formatting should be single backticks here. Running the genstdlib.jl script should result in this converting over to be equivalent to what you've written in the RST.